WalletWasabi.Fluent/Helpers/TextHelpers.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Linq;
+
+namespace WalletWasabi.Fluent.Helpers
+{
+	public static class TextHelpers
+	{
+		public static string AddSIfPlural(int n) => n > 1 ? "s" : "";
+
+		public static string CloseSentenceIfZero(params int[] counts) => counts.All(x => x == 0) ? "." : " ";
+
+		private static string ConcatNumberAndUnit(int n, string unit, char closingChar) => n > 0 ? $"{n} {unit}{AddSIfPlural(n)}{closingChar}" : "";
+
+		public static string TimeSpanToFriendlyString(TimeSpan time)
+		{
+			string result = "";
+
+			result += ConcatNumberAndUnit(time.Days, "day", ' ');
+			result += ConcatNumberAndUnit(time.Hours, "hour", ' ');
+			result += ConcatNumberAndUnit(time.Minutes, "minute", ' ');
```

Comment:
we probably should add more logic here for adding "and" before the last entry, like e.g.:
`1 day, 6 hours and 16 seconds`.

Reply:
we probably should use a library like https://github.com/Humanizr/Humanizer since this will not be the last time we will be dealing with human readable numbers... but i'll leave that to the higher ups to decide :)

Reply:
New format done. `Humanizer` looks interesting, @danwalmsley?